libs/designer/src/lib/ui/connections/dropzone.tsx

```diff
@@ -360,6 +363,13 @@ export const DropZone: React.FC<DropZoneProps> = memo(({ graphId, parentId, chil
                   </MenuItem>
                 </>
               )}
+              {UiInteractionsService()?.getAddButtonDropdownMenuItems?.({graphId, parentId, childId, preloadedOperations, dispatch}).map((item, index) => {
+                    return (
+                      <div key={index}>
+                        <CustomMenu item={item}/>
+                      </div>
+                    )
+                  })}
```

Comment:
nit: Please make sure the formatter is running over this. This doesn't looked aligned correctly